src/ui/molecules/stepper/stepper.scss

```diff
@@ -66,39 +66,37 @@ $colors: (
 }
 
 .okp4-stepper-body {
```

Comment:
Why `.okp4-stepper-body` is not inside `.okp4-stepper-main` as you have done for `.okp4-stepper-header`?
IMHO, the best way to use the `with-theme()` mixin is to wrap the whole selectors into the `.okp4-stepper-main` like this: 
```css
.okp4-stepper-main {
  @include with-theme() {
    ... /** all the others rules and selectors ho here **/
  }
}
```
If we have an issue with `stylelint` rules, we can see if it's relevant to increase the selector's depth.

Reply:
Yes that's right, from what I remember, it was a stylelint problem.

Reply:
Please, try it and tell me back

Reply:
It is indeed this error.

```
src/ui/molecules/stepper/stepper.scss
 40:15  ✖  Expected nesting depth to be no more than 6                                                       max-nesting-depth              
 40:15  ✖  Expected ".okp4-stepper-main .okp4-stepper-header .okp4-stepper-progress                          selector-max-compound-selectors
           .okp4-stepper-step-label.#{$color-key} div" to have no more than 4 compound selectors        
```

Reply:
I have increased the max compound selectors and max nesting depth